flask_common/documents.py

```diff
@@ -101,7 +107,16 @@ def delete(self, **kwargs):
         # delete only if already saved
         if self.pk:
             self.is_deleted = True
-            self.update(set__is_deleted=self.is_deleted)
+            update_kwargs = {
+                'set__is_deleted': self.is_deleted
+            }
+
+            # we don't want to update date_updated for deleted objects (in
+            # case they inherit from DocumentBase)
```

Comment:
It was causing some issues with the close.io tests, but I dug deeper and realized it was just the test assumptions that were wrong.